call-home/Cargo.toml

```diff
@@ -32,3 +32,4 @@ chrono = "0.4.23"
 rand = "0.8.5"
 tempfile = "3.3.0"
 humantime = "2.1.0"
+version-info = {path = "../dependencies/control-plane/utils/dependencies/version-info"}
```

Comment:
Add default-features = false or perhaps simply use utils crate from control-plane which is what other crates use.

Reply:
Using the utils crate.